test/utils/chrome.js

```diff
@@ -0,0 +1,265 @@
+/**
+ * @license Copyright 2016 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+import fs from 'fs'
+import path from 'path'
+import { execSync, execFileSync } from 'child_process'
+import isWsl from 'is-wsl'
+import consola from 'consola'
+import uniq from 'lodash/uniq'
+
+const newLineRegex = /\r?\n/
+
+/**
+ * This class is based on node-get-chrome
+ * https://github.com/mrlee23/node-get-chrome
+ * https://github.com/gwuhaolin/chrome-finder
+ */
+export default class ChromeDetector {
+  constructor() {
+    this.platform = isWsl ? 'wsl' : process.platform
+  }
+
+  detect(platform = this.platform) {
+    const handler = this[platform]
+    if (typeof handler !== 'function') {
+      throw new Error(`${platform} is not supported.`)
+    }
+    return this[platform]()[0]
+  }
+
+  darwin() {
+    const suffixes = [
+      '/Contents/MacOS/Chromium',
+      '/Contents/MacOS/Google Chrome Canary',
+      '/Contents/MacOS/Google Chrome'
+    ]
+    const LSREGISTER =
+      '/System/Library/Frameworks/CoreServices.framework' +
+      '/Versions/A/Frameworks/LaunchServices.framework' +
+      '/Versions/A/Support/lsregister'
+    const installations = []
+    const customChromePath = this.resolveChromePath()
+    if (customChromePath) {
+      installations.push(customChromePath)
+    }
+    execSync(
+      `${LSREGISTER} -dump` +
+        " | grep -i '(google chrome\\( canary\\)\\?|chromium).app$'" +
```

Comment:
@clarkdo Hi, just to notified you, if this command is invalid. I'm not sure why, but I can't get the desired result when using it. Just changed to the original " | grep -i 'google chrome\\( canary\\)\\?.app$'" and it worked.

Reply:
Thanks 😄 , fixed in https://github.com/nuxt/nuxt.js/commit/583e32a3ef0fc63fc76fa0f9d77994dea0382b25

Reply:
@clarkdo Nice 👍